manage-people/routes/PeopleRoutes.js

```diff
@@ -50,9 +50,25 @@ const peopleRoutes = (app) => {
                     res.status(200).send();
                 }
             });
-        });    
- 
+        });
+
+    router
+        .put('/:personId', (req, res) => {
+
+            Person.update({
+                guid: req.params.personId,
+                display_name: req.body.display_name,
+                authentication_identity: req.body.authentication_identity
+            }, function(err) {
+                if(err) {
+                   return res.status(500).send({message: 'Doh...'});
```

Comment:
you can return a 422 error, and the `err` value that you are getting from the db. 500 is too generic, and also in the frontend you won't see the cause of the error.